psake.cmd

```diff
@@ -1 +1,3 @@
+.\.nuget\nuget.exe install psake -Version 4.9.0 -OutputDirectory packages && ^
+.\.nuget\nuget.exe install xunit.runners -Version 1.9.2 -OutputDirectory packages && ^
```

Comment:
Curious why we have to do this vs installing it solution-wide...?

Reply:
So when you would do a fresh checkout, or simply delete the packages folder, and then run `psake` you will consistently get the following error:
![image](https://user-images.githubusercontent.com/8160149/224745744-9fd7cd81-df01-4962-8bf1-d5f35583bcef.png)

Then subsequent times it'll work no problem (which is likely why you may not have seen this error before). It looks to be a race condition only when the package isn't in the packages folder on first run.

I've got a new commit coming with a change to _default.ps1_ that will remove the line in _psake.cmd_ while eliminating the error on the initial checkout build.